app/models/notification.rb

```diff
@@ -2,8 +2,31 @@ class Notification < ActiveRecord::Base
   belongs_to :event
   # has_many :recipients
 
-  VALID_STATUSES = %w(pending active canceled complete expired)
-  validates :status, inclusion: { in: VALID_STATUSES }
+  enum status: [ :pending, :active, :canceled, :complete, :expired ]
```

Comment:
Moved to `enum`. Makes it easier to manage states, and also adds nifty methods. http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html